plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/MigrateKVMAsync.java

```diff
@@ -51,19 +76,19 @@ public MigrateKVMAsync(final LibvirtComputingResource libvirtComputingResource,
 
     @Override
     public Domain call() throws LibvirtException {
-        long flags = 1 << 0;
+        long flags = VIR_MIGRATE_LIVE;
 
         // set compression flag for migration, if libvirt version supports it
         if (dconn.getLibVirVersion() >= 1000003) {
-            flags |= 1 << 11;
+            flags += VIR_MIGRATE_COMPRESSED;
```

Comment:
Understanding how flags work, the or was a more readable syntax for me than the +=.

Reply:
For example, write as ``flags |= constant`